Source/WSTagsField.swift

```diff
@@ -303,6 +309,7 @@ public class WSTagsField: UIView {
 
     public func addTag(tag: String) {
         addTag(WSTag(tag))
+        tokenizeTextFieldText()
```

Comment:
Yeah, I realized my mistake. I mistook the function for something different. Look at the new(er) commit.